src/patient-search/patient-search.component.tsx

```diff
@@ -1,10 +1,96 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { match } from "react-router";
+import { debounce } from "lodash";
+import styles from "./patient-search.component.css";
+import { performPatientSearch } from "./patient-search.resource";
+import PatientSearchResults from "./patient-search-result/patient-search-result.component";
 
 export default function PatientSearch(props: PatientSearchProps) {
-  return <section>The patient search!</section>;
+  const searchTimeout = 400;
+  const [searchResults, setSearchResults] = useState([]);
+  const [emptyResult, setEmptyResult] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    if (searchTerm) {
+      performPatientSearch(searchTerm, "full").then(response => {
+        const { data } = response;
+        setSearchResults(data["results"]);
+        if (data["results"].length === 0) {
```

Comment:
nit: no need to use square brackets and quotes here:
```suggestion
        if (data.results.length === 0) {
```